internal/cluster/cluster_test.go

```diff
@@ -2258,6 +2258,62 @@ var _ = Describe("GenerateAdditionalManifests", func() {
 	})
 })
 
+var _ = Describe("Deregister inactive clusters", func() {
+	var (
+		ctrl          *gomock.Controller
+		ctx           = context.Background()
+		db            *gorm.DB
+		state         API
+		c             common.Cluster
+		eventsHandler events.Handler
+		mockMetric    *metrics.MockAPI
+		dbName        = "permanently_delete_clusters"
+		mockS3Api     *s3wrapper.MockAPI
+	)
+
+	registerCluster := func() common.Cluster {
```

Comment:
there is already a function that can create a cluster, not need to duplicate https://github.com/openshift/assisted-service/blob/master/internal/bminventory/inventory_test.go#L859-L861

Reply:
Why was it resolved? 
I see another test that have exactly the same function `Permanently delete clusters`